src/content/docs/en/core-concepts/endpoints.mdx

```diff
@@ -127,8 +126,20 @@ export async function get({ params }) {
   });
 }
 ```
+
 This will respond to any request that matches the dynamic route. For example, if we navigate to `/helmet.json`, `params.id` will be set to `helmet`. If `helmet` exists in the mock product database, the endpoint will use create a `Response` object to respond with JSON and return a successful [HTTP status code](https://developer.mozilla.org/en-US/docs/Web/API/Response/status). If not, it will use a `Response` object to respond with a `404`.
 
+In **SSR** mode certain providers require the `Content-Type` header to return an image. In this case, use a `Response` object to specify a `headers` property like in the example above.
```

Comment:
```suggestion
In SSR mode, certain providers require the `Content-Type` header to return an image. In this case, use a `Response` object to specify a `headers` property. For example, to produce a binary `.png` image:
```
This looks good, and well-placed, thank you! I've made some slight adjustments because now this is in the SSR section, I don't think SSR needs to be bolded here. And, someone may not have read the above section on static endpoints, so I think we can keep the description self-contained.

I'm just going to commit this so we can go ahead and merge it!  :raised_hands: